cmd/configschema/go.mod

```diff
@@ -62,7 +62,6 @@ require (
 	github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace v1.8.5 // indirect
 	github.com/GoogleCloudPlatform/opentelemetry-operations-go/internal/resourcemapping v0.32.5 // indirect
 	github.com/Microsoft/go-winio v0.5.2 // indirect
-	github.com/Microsoft/hcsshim v0.9.2 // indirect
```

Comment:
Did you just manually remove this? Or did you run go mod tidy? I believe @codeboten ran a go mod tidy recently, which should have addressed this, if removing is the right thing to do...

Reply:
@codeboten, shouldn't your recent "go mod tidy" PRs have addressed this?